mmv1/templates/terraform/examples/bigquery_reservation_capacity_commitment_no_id.tf.erb

```diff
@@ -0,0 +1,13 @@
+resource "google_bigquery_capacity_commitment" "<%= ctx[:primary_resource_id] %>" {
```

Comment:
Also, indentations are weird in this file and should be two white spaces.

Reply:
Fixed